modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk/query_processor.clj

```diff
@@ -164,14 +168,26 @@
     :type/DateTime       :datetime
     nil))
 
-(defmethod temporal-type (class Field)
-  [{base-type :base_type, effective-type :effective_type, database-type :database_type}]
-  (case database-type
+(defn- database-type->temporal-type [database-type]
+  (condp = (some-> database-type str/upper-case)
     "TIMESTAMP" :timestamp
```

Comment:
It says here they're working on an interval type that someone's probably going to ask us for whenever they finish it

https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#interval_type

Reply:
FWIW Postgres has interval types and we don't support those yet either